tools/accuracy_checker/accuracy_checker/annotation_converters/ms_asl_continuous.py

```diff
@@ -0,0 +1,155 @@
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+from ..utils import read_txt, check_file_existence
+from ..representation import ClassificationAnnotation
+from ..data_readers import ClipIdentifier
+from ..config import PathField, NumberField
+
+from .format_converter import BaseFormatConverter, ConverterReturn
+
+
+class RawFramesSegmentedRecord:
+    def __init__(self, row):
+        self._data = row
+
+        assert self.video_num_frames > 0
+        assert self.num_frames > 0
+        assert self.fps > 0
+        assert self.label >= 0
+        assert self.clip_start >= self.video_start >= 0
+        assert self.video_end >= self.clip_end >= 0
+
+    @property
+    def path(self):
+        return self._data[0]
+
+    @property
+    def label(self):
+        return int(self._data[1])
+
+    @property
+    def clip_start(self):
+        return int(self._data[2])
+
+    @property
+    def clip_end(self):
+        return int(self._data[3])
+
+    @property
+    def video_start(self):
+        return int(self._data[4])
+
+    @property
+    def video_end(self):
+        return int(self._data[5])
+
+    @property
+    def fps(self):
+        return float(self._data[6])
+
+    @property
+    def num_frames(self):
+        return self.clip_end - self.clip_start
+
+    @property
+    def video_num_frames(self):
+        return self.video_end - self.video_start
+
+
+class MSASLContiniousConverter(BaseFormatConverter):
+    __provider__ = 'continuous_clip_action_recognition'
+    annotation_types = (ClassificationAnnotation, )
+
+    @classmethod
+    def parameters(cls):
+        params = super().parameters()
+        params.update({
+            'annotation_file': PathField(description="Path to annotation file."),
+            'data_dir': PathField(is_directory=True, description="Path to data directory."),
+            'out_fps': NumberField(
+                value_type=float, optional=True, min_value=1, default=15, description="Output FPS."
+            ),
+            'clip_length': NumberField(
+                value_type=int, optional=True, min_value=1, default=16, description="Clip length."
+            )
+        })
+
+        return params
+
+    def configure(self):
+        self.annotation_file = self.get_value_from_config('annotation_file')
+        self.data_dir = self.get_value_from_config('data_dir')
+        self.out_fps = self.get_value_from_config('out_fps')
+        self.clip_length = self.get_value_from_config('clip_length')
+
+    def convert(self, check_content=False, progress_callback=None, progress_interval=100, **kwargs):
+        records = self.load_annotations(self.annotation_file)
+
+        annotations = []
+        num_iterations = len(records)
+        content_errors = None if not check_content else []
+        for record_idx, record in enumerate(records):
+            if progress_callback is not None and record_idx % progress_interval:
```

Comment:
record_idx % progress_interval will be true when result of the division will have a remainder, right?
If yes, we need reverse action (log progress only when n steps passed and remainder is zero).

Reply:
Yes, thanks. Fixed.